src/ssl/ensemble.py

```diff
@@ -1,17 +1,20 @@
-import numpy as np
 from statistics import mode
 
-from src.flexcon import FlexConC
+import numpy as np
+
+from src.ssl import flexcon
 
 
-class Ensemble():
+class Ensemble:
     """
     Classe responsável por criar um cômite de classificadores e implementar
     seus métodos
     """
 
-    def __init__(self) -> None:
+    def __init__(self, ssl_algorithm: callable, **kwargs):
         self.ensemble = []
+        self.ssl_algorithm = ssl_algorithm
+        self.params = kwargs
 
     def add_classifier(self, classifier):
```

Comment:
```suggestion
    def add_classifier(self, classifier) -> None:
```

colocar o tipo o argumento "classifier".

Reply:
feito